components/admin_console/push_settings.jsx

```diff
@@ -131,23 +131,36 @@ export default class PushSettings extends AdminSettings {
+                    defaultMessage='Download <linkIOS>Mattermost iOS app</linkIOS> from iTunes. Download <linkAndroid>Mattermost Android app</linkAndroid> from Google Play. Learn more about the <linkHPNS>Mattermost Hosted Push Notification Service</linkHPNS>.'
+                    values={{
+                        linkIOS: (msg: React.ReactNode) => (<a href='https://mattermost.com/mattermost-ios-app/' referrer='noreferrer' target='_blank'>{msg}</a>),
+                        linkAndroid: (msg: React.ReactNode) => (<a href='https: //mattermost.com/mattermost-android-app/' referrer='noreferrer' target='_blank'>{msg}</a>),
+                        linkHPNS: (msg: React.ReactNode) => (<a href='https://docs.mattermost.com/deploy/mobile-hpns.html]' referrer='noreferrer' target='_blank'>{msg}</a>),
+                    }}
                 />
             );
         } else if (this.state.pushNotificationServerType === PUSH_NOTIFICATIONS_MTPNS) {
             pushServerHelpText = (
                 <FormattedMarkdownMessage
                     id='admin.email.mtpnsHelp'
-                    defaultMessage='Download [Mattermost iOS app](!https://mattermost.com/mattermost-ios-app/) from iTunes. Download [Mattermost Android app](!https://mattermost.com/mattermost-android-app/) from Google Play. Learn more about the [Mattermost Test Push Notification Service](!https://docs.mattermost.com/deploy/mobile-hpns.html).'
+                    defaultMessage='Download <linkIOS>Mattermost iOS app</linkIOS> from iTunes. Download <linkAndroid>Mattermost Android app</linkAndroid> from Google Play. Learn more about the <linkHPNS>Mattermost Hosted Push Notification Service</linkHPNS>.'
+                    values={{
+                        linkIOS: (msg: React.ReactNode) => (<a href='https://mattermost.com/mattermost-ios-app/' referrer='noreferrer' target='_blank'>{msg}</a>),
+                        linkAndroid: (msg: React.ReactNode) => (<a href='https: //mattermost.com/mattermost-android-app/' referrer='noreferrer' target='_blank'>{msg}</a>),
+                        linkHPNS: (msg: React.ReactNode) => (<a href='https://docs.mattermost.com/deploy/mobile-hpns.html]' referrer='noreferrer' target='_blank'>{msg}</a>),
+                    }}
                 />
             );
         } else {
             pushServerHelpText = (
-                <FormattedMarkdownMessage
+                <FormattedMessage
                     id='admin.email.easHelp'
-                    defaultMessage='Learn more about compiling and deploying your own mobile apps from an [Enterprise App Store](!https://docs.mattermost.com/).'
+                    defaultMessage='Learn more about compiling and deploying your own mobile apps from an <link>Enterprise App Store</link>.'
+                    values={{
+                        link: (msg: React.ReactNode) => (<a href='https://docs.mattermost.com/' referrer='noreferrer' target='_blank'>{msg}</a>),
```

Comment:
is this the correct link for learning more about compiling and deploying the mobile apps @cwarnermm ?